v2/validator.go

```diff
@@ -33,6 +33,7 @@ type urlValidationResponse struct {
 	Cors         bool          `json:"cors"`
 	ContentType  bool          `json:"contentType"`
 	CertValid    bool          `json:"certValid"`
+	ValidatedJson interface{} `json:"validatedJson,omitempty"`
```

Comment:
Your horizontal alignment seems to be off 🙂

Reply:
wops, forgot to run go fmt!
One day i will add it to my IDE, really! :joy: